### DISCUSSION TITLE
Docker Photoprism error after server reboot

### DISCUSSION THREAD

**Discussion by Important96** [photoprism/photoprism#4090] [Q&A]
I run Photoprism on Truenas Scale.

After reboot, I get this message and the software won't run

`2024-03-03 22:24:56.136073+00:00 started 231128 as root (amd64-prod)
2024-03-03 22:24:56.158928+00:00 init: updating filesystem permissions
2024-03-03 22:24:56.158957+00:00 PHOTOPRISM_DISABLE_CHOWN="true" disables permission updates
`
That's all in logs.

I've checked and all user permissions look ok. 
Found something online but won't work for me.
Thanks.

**Comment by graciousgrey** [comment]
What error do you get? The logs you posted look normal to me.

You might get some more insights when setting the log level to trace.

https://docs.photoprism.app/getting-started/troubleshooting/logs/#__tabbed_1_3
